src/components/operations-components/filter-elements.js

```diff
@@ -0,0 +1,15 @@
+import { useOperations } from "../../context/operations-context"
+
+const FilterElements = ({label}) => {
+
+    const {operationsState : {labelFilters}} = useOperations()
+
+    return(
+        <div className="flex gap-m align-center">
+            <input type = "checkbox" checked = {labelFilters[label] === true}/>
+            <p>{label}</p>
```

Comment:
instead of giving `labelFilters[label] === true` you can directly give `labelFilters[label]`